python/packages/ai/teams/ai/prompts/completion_config.py

```diff
@@ -71,6 +73,7 @@ class CompletionConfig:
     stop_sequences: Optional[List[str]] = None
     temperature: float = 0
     top_p: float = 0
+    data_sources: Optional[list[object]] = None
```

Comment:
can we update to uppercase List (and also in comment)

Reply:
fixed